pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
 requires = [
-  "setuptools >= 64.0.0",  # minimum for PEP 660 editable install
+  "setuptools >= 77.0.0",  # minimum for PEP 639 which changes the license file format in project.toml
```

Comment:
Given that version 77.0.0 was only released on March 19 2025 I think this is perhaps too much of a change. I would *hope* that we can use earlier versions even with these changes (perhaps leaving in the license classifier below so that users of older setuptools do not lose information)?

Although maybe the older versions will fall over with the `license` setting? I am too tired / annoyed about all this to go and look.

Reply:
I'm going to experiment what the minimum version is, now that I've identified all that we need to do to be compatible with the *newest* version. Expect to see a few CI runs, because that way I'll see that it works for linux, too, and for all the different setups that we use.

Reply:
The change in the license format done here was introduced in 77.0 and is not compatible with previous versions. I'll need to revert that and pin to "<77.0.0" and then we can have a follow-up at some later point and change that.